.travis.yml

```diff
@@ -5,7 +5,7 @@ branches:
 matrix:
   include:
     - language: rust
-      rust: stable
+      rust: stable-2018-06-21
```

Comment:
I'd prefer if we kept being compatible with latest `stable`

Reply:
the problem with that would be that the rustfmt check on travis (`cargo fmt --all -- --write-mode=diff`) then sometimes breaks the build from one run to the next because rustfmt rules have changed slightly. had this happen a bunch of times and it's annoying. i'll add a comment explaining it

Reply:
comment added